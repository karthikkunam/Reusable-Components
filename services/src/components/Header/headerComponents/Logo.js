import React from 'react';
import PropTypes from 'prop-types';

const Logo = props => {
    const {logoClassName} = props;
    return (
        <div className="navbar-brand header-logo-container">
            <span id="header-logo" className={logoClassName} alt="Seven eleven logo"></span>
        </div>
    );
};

Logo.propTypes = {
    logoClassName: PropTypes.string,
};
export default Logo;