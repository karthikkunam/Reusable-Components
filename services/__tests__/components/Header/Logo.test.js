import React from 'react';
import Logo from '../../../src/components/Header/headerComponents/Logo';
import { shallow } from 'enzyme';

describe('Logo Component', () => {
  it('renders correctly', () => {
    const props = {
      logoClassName: 'navbar-brand-logo header-logo img-fluid',
    };
    const component = shallow(<Logo props={props} />);
    expect(component).toMatchSnapshot();
  });
});
