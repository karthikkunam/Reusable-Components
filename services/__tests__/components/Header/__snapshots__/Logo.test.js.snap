// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component renders correctly 1`] = `
<div
  className="navbar-brand header-logo-container"
>
  <span
    alt="Seven eleven logo"
    id="header-logo"
  />
</div>
`;
